tests/backup/backup_upgrade_test.go

```diff
@@ -612,6 +612,9 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", func() {
 			restoreSingleNSBackupInVariousWaysTask := func(index int, namespace string) {
 				nextScheduleBackupName, err := GetNextScheduleBackupName(singleNSScheduleNames[index], time.Duration(intervalInMins), ctx)
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching next schedule backup name of schedule named [%s]", singleNSScheduleNames[index]))
+				appContextsToBackup := FilterAppContextsByNamespace(srcClusterContexts, []string{namespace})
+				err = backupSuccessCheckWithValidation(ctx, nextScheduleBackupName, appContextsToBackup, orgID, maxWaitPeriodForBackupCompletionInMinutes*time.Minute, 30*time.Second)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of success of next single namespace schedule backup [%s] of schedule %s", nextScheduleBackupName, singleNSScheduleNames[index]))
```

Comment:
Can we replace CreateBackup with CreateBackupWithValidation?

Reply:
and CreateRestore with CreateRestoreWithValidation?

Reply:
At which line no are you asking to use createBackupWithValidation?
@kphalgun-px

Reply:
Everywhere in the test case.

Reply:
Yes will change that, but we still will need to use backupSuccessCheckWithValidation for next schedule backups

Reply:
Right. My bad.

Reply:
@kphalgun-px 
changed CreateRestore with CreateRestoreWithValidation and CreateBackup with CreateBackupWithValidation

Reply:
Thank you.